climada/hazard/tc_tracks_synth.py

```diff
@@ -342,6 +439,21 @@ def _one_rnd_walk(track, nb_synth_tracks, max_shift_ini, max_dspeed_rel, max_ddi
         i_track.attrs['sid'] = f"{i_track.attrs['sid']}_gen{i_ens + 1}"
         i_track.attrs['id_no'] = i_track.attrs['id_no'] + (i_ens + 1) / 100
         i_track = i_track.isel(time=slice(None, last_idx))
+        if adjust_intensity:
+            # compute minimum pressure occurring on or after each timestep
```

Comment:
@bguillod This is where I'd implement the fitting of MSW and radii to p_central.